package-lock.json

```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
```

Comment:
This is what happens if I run `npm install` with npm 5.6.0 :roll_eyes:

Reply:
yeah, renovate seems to have some kind of non-standard setting for creating/updating package-lock.json that I've thus far been unable to reproduce locally.